docs/running/cwl.rst

```diff
@@ -38,3 +86,145 @@ multiple files, it may be useful. For example, if you want to run a CWL workflow
 samples inputs, it could look something like:
 
 .. literalinclude:: ../../src/toil/test/docs/scripts/tutorial_cwlexample.py
+
+Toil & CWL Tips
+---------------
+
+See logs for just one job (by using the full log file, requires knowing the job's toil-generated ID):
+::
+
+    cat cwltoil.log | grep jobVM1fIs
+
+Grep for full commands from toil logs
+
+This gives you a more concise view of the commands being run (note that this information is only available from
+Toil when running with `--logDebug`, or when using the `pipeline_submit` script with `--logLevel DEBUG`):
+::
+
+    pcregrep -M "\[job .*\.cwl.*$\n(.*        .*$\n)*" cwltoil.log
+              ^allows for multiline matching
+
+
+Find Bams that have been generated for specific step while pipeline is running:
+::
+
+    find . | grep -P '^./out_tmpdir.*_MD\.bam$'
+
+
+Print graphical representation of the workflow:
+::
+
+    cwltool --print-dot workflows/innovation_pipeline.cwl | dot -Tpdf -o innovation_pipeline.pdf && open innovation_pipeline.pdf
```

Comment:
I think `open` is macOS specific? `xdg-open` is more generic

Reply:
👍